src/components/intro/index.scss

```diff
@@ -4,7 +4,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  max-width: 60%;
+  max-width: 90%;
```

Comment:
Now that the logo is way smaller, the intro feels big design-wise (definitely not a designer btw 😅)

I tried leaving the max-width at 60%, and reducing the font-size to 1rem, and it looks better to me. Feedback welcome.

Reply:
70% and `1.25rem` also look good 🤔 Let's think a bit more and come back to it

Reply:
For me, 70% and 1.25rem looks better 😄 . I'll change it if that's ok

Reply:
All good!

Reply:
Done!